### PR TITLE
Misc. bugfixes including memory errors in typemaps, repaired and new …

### PR DESCRIPTION
**Pull request by markshowalter**
…unit tests. Also a new 0.0.1 version number.

### CODE REVIEW

**Comment by fyellin** [comment]
I've created a test pypi release of Mark's changes.

```pip install -i https://test.pypi.org/simple/ cspyce==2.0.5a0```

Everything except MacOS Python 3.10 is there.  (That failed for an internal github reason, and I'm rebuilding.  Probably will be available in about 10 minutes.)

**Review by rfrenchseti — CHANGES_REQUESTED** [review]
Are `cspyce0.py` and `cspyce0_info.py` and `vectorize.i` supposed to be checked in? They are automatically generated and weren't checked in before.

**Review by rfrenchseti — COMMENTED** [review]
This branch passes all oops unit tests and my navigation software runs without crashing.

File: cspyce/__init__.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ################################################################################
 # cspyce/__init__.py
 ################################################################################
-# SPYCE OVERVIEW
+# CSPYCE OVERVIEW
 #
-# Spyce is a Python module that provides an interface to the CSPICE library. It
+# Cspyce is a Python module that provides an interface to the CSPICE library. It
```
Comment (rfrenchseti) [inline cspyce/__init__.py:6]: cspyce should be lower case

**Review by rfrenchseti — CHANGES_REQUESTED** [review]

**Comment by rfrenchseti** [comment]
These changes pass OOPS and my navigation software

File: cspyce/__init__.py (hunk 1 above)
Comment (markshowalter) [reply cspyce/__init__.py:6]: I have no opinion on this. Change it if you feel strongly.

**Review by rfrenchseti — APPROVED** [review]

**Pull request merged**